### PR TITLE
Small addition for new output files/formats and assess structure predictions

### PR DESCRIPTION
**Pull request by fsimkovic**

### CODE REVIEW

File: conkit/misc/selectalg.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        list
+           The decoy indices to keep
+        list
+           The decoy indices to throw
+
+        """
+        data = SubselectionAlgorithm._numpify(data)
+        data_scaled = data / np.mean(data)
+        keep = np.where(data_scaled >= cutoff)[0]
+        throw = np.where(data_scaled < cutoff)[0]
```
Comment (hlasimpk) [inline conkit/misc/selectalg.py]: Could replace this with a call to the cutoff function above which seems to do the same thing.

File: conkit/misc/selectalg.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        else:
+            return np.asarray(data)
+
+    @staticmethod
+    def cutoff(data, cutoff=0.287):
+        """A cutoff-defined subselection algorithm
+
+        Description
+        -----------
+        This algorithm removes a decoy, if its score is l
```
Comment (hlasimpk) [inline conkit/misc/selectalg.py]: less is split over 2 lines

File: conkit/misc/selectalg.py [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+        throw = np.where(data < cutoff)[0]
+        return keep.tolist(), throw.tolist()
+
+    @staticmethod
+    def linear(data, cutoff=0.5):
+        """A linearly-defined subselection algorithm
+
+        Description
+        -----------
+        This algorithm removes the worst 500 decoys.
```
Comment (hlasimpk) [inline conkit/misc/selectalg.py]: Does this algorithm do this? Or does it just remove the worst 50%?

File: conkit/misc/selectalg.py [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+        -------
+        list
+           The decoy indices to keep
+        list
+           The decoy indices to throw
+
+        """
+        data = SubselectionAlgorithm._numpify(data)
+        keep = np.where(data >= 0)[0]
+        throw = np.where(data < 0)[0]
```
Comment (hlasimpk) [inline conkit/misc/selectalg.py]: Again this could be replaced with a call to cutoff

File: conkit/misc/selectalg.py (hunk 3 above)
Comment (fsimkovic) [reply conkit/misc/selectalg.py]: good spot, thanks

File: conkit/misc/selectalg.py (hunk 2 above)
Comment (fsimkovic) [reply conkit/misc/selectalg.py]: good spot, thanks

File: conkit/misc/selectalg.py (hunk 4 above)
Comment (fsimkovic) [reply conkit/misc/selectalg.py]: Good spot, thanks

File: conkit/misc/selectalg.py (hunk 1 above)
Comment (fsimkovic) [reply conkit/misc/selectalg.py]: Good spot, thanks

**Pull request merged**